back/spec/factories/activities.rb

```diff
@@ -63,6 +68,21 @@
       action { 'idea_disliked' }
     end
 
+    factory :initiative_published_activity do
+      association :item, factory: :initiative
+      action { 'published' }
+    end
+
+    factory :initiative_liked_activity do
+      association :item, factory: :reaction
+      action { 'initiative_liked' }
+    end
+
+    factory :initiative_disliked_activity do
+      association :item, factory: :dislike
+      action { 'initiative_disliked' }
+    end
```

Comment:
Aren't the activities created by those factories a bit inconsistent, since the reaction/dislike will by associated to an idea by default instead of an initiative?

Reply:
Yes, good catch. Will fix.